src/server/mock-data/json/toerisme.json

```diff
@@ -0,0 +1,25 @@
+{
+  "content": [
+    {
+      "city":"Amsterdam",
+      "houseLetter":null,
+      "houseNumber":"16",
+      "houseNumberExtension":null,
+      "postalCode":"1014AW",
+      "registrationNumber":"0363 E7B8 B042 8A92 37E5",
+      "shortName":"Schakelstraat",
+      "street":"Schakelstraat"
+      },
+      {
```

Comment:
Wanneer je een item dupliceert, kun je dan de inhoud ook aanpassen zodat we niet met dubbele identifiers werken en beter kunnen zien dat er verschillende soorten inhoud aanwezig kunnen zijn.